architecture/audience.rst

```diff
@@ -0,0 +1,14 @@
+Audience
+========
+
+The audience for this chapter on architecture is release engineering contributors (including employees, of course).
```

Comment:
Pedantic level: 11 -- maybe put `(releng)` in here as we stop referring to 'release engineering' after this point.